packages/gatsby-source-nacelle/src/services/nacelleClient.js

```diff
@@ -1,49 +1,19 @@
-const NacelleClient = require('@nacelle/client-js-sdk').default;
+const NacelleClient = require('@nacelle/storefront-sdk').default;
```

Comment:
Can we delete this whole file? Since we're only using the passed in client, I'm not sure that we need this service anymore.

Reply:
Good point, removed!